src/claimtrie.cpp

```diff
@@ -1130,91 +669,71 @@ bool CClaimTrie::ReadFromDisk(bool check)
         LogPrintf("%s: Couldn't read the best block's hash\n", __func__);
     if (!db.Read(CURRENT_HEIGHT, nCurrentHeight))
         LogPrintf("%s: Couldn't read the current height\n", __func__);
+
     setExpirationTime(Params().GetConsensus().GetExpirationTime(nCurrentHeight-1));
 
-    boost::scoped_ptr<CDBIterator> pcursor(const_cast<CDBWrapper*>(&db)->NewIterator());
+    typedef std::map<std::string, CClaimTrieNode, nodenamecompare> trieNodeMapType;
 
-    pcursor->SeekToFirst();
-    while (pcursor->Valid())
-    {
-        std::pair<char, std::string> key;
-        if (pcursor->GetKey(key))
-        {
-            if (key.first == TRIE_NODE)
-            {
-                CClaimTrieNode* node = new CClaimTrieNode();
-                if (pcursor->GetValue(*node))
-                {
-                    if (!InsertFromDisk(key.second, node))
-                    {
-                        return error("%s(): error restoring claim trie from disk", __func__);
-                    }
-                }
-                else
-                {
-                    return error("%s(): error reading claim trie from disk", __func__);
-                }
-            }
+    trieNodeMapType nodes;
+    if(!db.seekByKey(nodes)) {
+        return error("%s(): error reading claim trie from disk", __func__);
+    }
+
```

Comment:
Upon fresh start (with empty data directory), there won't be any nodes to load, so this will cause lbrycrdd to exit. See
https://github.com/lbryio/lbrycrd/blob/master/src/init.cpp#L1286